gdcdictionary/schemas/_definitions.yaml

```diff
@@ -44,176 +44,177 @@ to_many:
+    - error
+    - released
+    - suppressed
+    - redacted
 
 data_file_error_type:
-    term:
-        $ref: "_terms.yaml#/data_file_error_type"
-    enum:
-        - file_size
-        - file_format
-        - md5sum
+  $ref: "_terms.yaml#/data_file_error_type"
+  enum:
+    - file_size
+    - file_format
+    - md5sum
 
 state:
-    term:
-        $ref: "_terms.yaml#/state"
-    default: validated
-    downloadable:
-        - uploaded
-        - md5summed
-        - validating
-        - validated
-        - error
-        - invalid
-        - released
-    public:
-        - live
-    oneOf:
-        - enum: #This list covers legacy data
-            - uploading
-            - uploaded
-            - md5summing
-            - md5summed
-            - validating
-            - error
-            - invalid
-            - suppressed
-            - redacted
-            - live
-        - enum: #This list covers all future data 1/15/16
-            - validated
-            - submitted
-            - released
+  $ref: "_terms.yaml#/state"
+  default: validated
+  downloadable:
+    - uploaded
+    - md5summed
+    - validating
+    - validated
+    - error
+    - invalid
+    - released
+  public:
+    - live
+  oneOf:
+    - enum: #This list covers legacy data
+      - uploading
+      - uploaded
+      - md5summing
+      - md5summed
+      - validating
+      - error
+      - invalid
+      - suppressed
+      - redacted
+      - live
+    - enum: #This list covers all future data 1/15/16
+      - validated
+      - submitted
+      - released
 
 file_state:
-    term:
-        $ref: "_terms.yaml#/file_state"
-    default: registered
-    enum:
-        - registered
-        - uploading
-        - uploaded
-        - validating
-        - validated
-        - submitted
-        - processing
-        - processed
-        - released
-        - error
-        - deleted
+  $ref: "_terms.yaml#/file_state"
+  default: registered
+  enum:
+    - registered
+    - uploading
+    - uploaded
+    - validating
+    - validated
+    - submitted
+    - processing
+    - processed
+    - released
+    - error
+    - deleted
 
 qc_metrics_state:
-    term:
-      $ref: "_terms.yaml#/qc_metric_state"
-    enum:
-        - FAIL
-        - PASS
-        - WARN
-        - Unknown
-        - Not Reported
+  $ref: "_terms.yaml#/qc_metric_state"
+  enum:
+    - FAIL
+    - PASS
+    - WARN
+    - Unknown
+    - Not Reported
 
 project_id:
-    type: string
-    term:
-        $ref: "_terms.yaml#/project_id"
+  $ref: "_terms.yaml#/project_id"
+  type: string
 
 data_file_properties:
   $ref: "#/ubiquitous_properties"
+
   file_name:
     $ref: "#/file_name"
+
   file_size:
     $ref: "#/file_size"
+
   md5sum:
     $ref: "#/md5sum"
+
   file_state:
     $ref: "#/file_state"
+
   error_type:
     $ref: "#/data_file_error_type"
+
   state_comment:
+    description: Optional comment about why the file is in the current state, mainly for invalid state.
```

Comment:
This an example of a single-line, plain string. In the previous version, all line breaks were preserved.  In this version, all line-breaks are removed.